mkosi/__init__.py

```diff
@@ -108,8 +108,8 @@ def mount_base_trees(context: Context) -> Iterator[None]:
                 extract_tar(path, d, sandbox=context.sandbox)
                 bases += [d]
             elif path.suffix == ".raw":
-                run(["systemd-dissect", "-M", path, d])
-                stack.callback(lambda: run(["systemd-dissect", "-U", d]))
```

Comment:
These are not the same as `-M` and `-U`, please see the manpage.

Reply:
Should I then just drop the commit or add `--mkdir` and `--rmdir`?

Reply:
Add --mkdir and --rmdir